packages/@ember/-internals/runtime/lib/mixins/array.js

```diff
@@ -652,12 +651,24 @@ const ArrayMixin = Mixin.create(Enumerable, {
     - `index` is the current index in the iteration.
     - `array` is the array itself.
 
-    It should return `true` to include the item in the results, `false`
-    otherwise.
+    All parameters are optional. The function should return `true` to include the item
+    in the results, and `false` otherwise.
 
-    Note that in addition to a callback, you can also pass an optional target
-    object that will be set as `this` on the context. This is a good way
-    to give your iterator function access to the current object.
+    Example:
+
+    ```javascript
+    function isAdult(person) {
+      return person.age > 18;
+    };
+
+    const people = Ember.A([{ name: 'John', age: 14 }, { name: 'Joan', age: 45 }]);
```

Comment:
Ember uses a [liberal let](https://madhatted.com/2016/1/25/let-it-be) convention, so these should be `let`s.